### DISCUSSION TITLE
[Suggestions] Adventure map, improve the edges of the adventure map

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#7000] [Ideas]
To avoid having "cut" sprites, especially at the top of the adventure map, I think we should add a border all around the adventure map that would match what is present on the adventure map to have a more aesthetic visual by allowing all sprites (and other decors) to be displayed entirely..

In original game:

Boat sprite cut:
![Edge adventure map 001](https://user-images.githubusercontent.com/43583994/231844160-33c4d6d3-f2b6-4397-8447-5209bb98726d.png)

Hero sprite cut:
![Edge adventure map 004](https://user-images.githubusercontent.com/43583994/231844172-a4d14bd7-2975-4ee8-b85d-da8934cba108.png)

**Comment by Mr-Bajs** [comment]
In fheroes2 they are not cut att the top border
![border](https://user-images.githubusercontent.com/93934125/231848253-cc85f15f-fcce-4929-b80e-c6b8dfe9ec8a.png)

**Reply by LeHerosInconnu** [reply]
Hello @Mr-Bajs,

> In fheroes2 they are not cut att the top border
> ![border](https://user-images.githubusercontent.com/93934125/231848253-cc85f15f-fcce-4929-b80e-c6b8dfe9ec8a.png)

Yes, but it's not very beautiful, not very aesthetic. :)

**Reply by zenseii** [reply]
Personally I think extending above the upper border is fine but the lateral one is not very nice. He's almost falling outside of the map :P
